### DISCUSSION TITLE
How to I create a custom week?

### DISCUSSION THREAD

**Discussion by squeed-lol** [KadeArchive/Kade-Engine#212] [Q&A] [answered]
I want to create a custom week but I dont know how. I want to be able to do it just through the assets folder instead of having to go through the source code, but if i cant thats fine

**Comment by theDetourist** [comment, marked as answer]
Quoting what I said @ #189:

> First, [watch this](https://www.youtube.com/watch?v=ac94p0zqqYM).
> Then, tell us how much progress you've managed to make based on that.
> 
> And if that's not enough, I'm sure a quick Youtube or Google search shall give you fine results.

**Reply by Fefe543** [reply]
i don't want to be rude, but the coding source CHANGES, and it's hard to make it.
2nd: google and youtube don't have answers sometimes, only your videos

**Reply by theDetourist** [reply]
Maybe to really specific cases but if you really want to create something new instead of simply replacing files to create a custom week, you'll have to dive into critical thinking and Haxe coding. We can only guide you, but you're the one that has to do the heavy lifting.

I learned Haxe in more or less a month, you can do it too if you really want it.

**Reply by Fefe543** [reply]
i did learn from your video, but still has problems tho. Aswell the source codes changes.
anyways bye have a good day

**Reply by isakube** [reply]
https://github.com/KadeDev/Kade-Engine/discussions/226
